### DISCUSSION TITLE
Ground item highlighting item clashes with "filter based on quantity"

### DISCUSSION THREAD

**Discussion by korvl38** [runelite/runelite#17860] [Ideas]
Example: I want coins<500 not shown, but also want coins above 500 highlighted. If I set coins<500 but also highlight coins, then 1 coin will override the filter.

I've always had coins highlighted and I remember setting my filter for coins under certain amount working perfectly to get rid of small amounts of coins on the floor.

**Comment by YvesW** [comment]
Can you not put ``coins>3000`` in the highlight box? Please note that I've not checked the code, nor tested this ingame.

**Reply by korvl38** [reply]
Wow that was easy, thanks a lot! Works perfectly. It seems like the opposite way doesn't work but I have zero understand of coding
Works: Highlight "coins>500", hide "coins"
Doesn't work:  Highlight "coins", hide "coins<500"
